config.schema.yaml

```diff
@@ -0,0 +1,20 @@
+---
+appliance:
+  type: object
+  properties:
+    default:
```

Comment:
I don't _think_ this is right. I think it works if you have just one `default` appliance setting, but it won't work if you declare multiple appliance settings. I think it needs to look more like this: https://github.com/StackStorm-Exchange/stackstorm-mysql/blob/master/config.schema.yaml